api/envoy/config/health_checker/redis/v3/redis.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.config.health_checker.redis.v3;
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.config.health_checker.redis.v3";
+option java_outer_classname = "RedisProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Redis]
+// Redis health checker :ref:`configuration overview <config_health_checkers_redis>`.
+// [#extension: envoy.health_checkers.redis]
+
+message Redis {
+  option (udpa.annotations.versioning).previous_message_type =
+      "envoy.config.health_checker.redis.v2.Redis";
```

Comment:
Actually, wait a second; I asked that we avoid having these; otherwise we will see rejection of of the existing v2 protos that are part of v3 effectively.

Reply:
really sorry. i missed it somehow. let me recheck it now.

Reply:
i am bit confused here , since i am new to this. i am curious to know if your mark v2 proto as FROZEN then how come it will be part of v3 ?  ideally both should be ACTIVE , right ? just for my understanding !!!

Reply:
You will need to have the proto format tools allowlist (i.e. special case) these files and skip enforcement of the normal active/frozen rules.

Reply:
Ok. Got it !!!

Reply:
i shall push the code ASAP , trying  first in my local env. sorry for delay !!!